src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Global.cs

```diff
@@ -5,6 +5,8 @@
 
 [assembly: InternalsVisibleTo(@"Microsoft.TemplateEngine.Orchestrator.RunnableProjects.UnitTests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100f33a29044fa9d740c9b3213a93e57c84b472c84e0b8a0e1ae48e67a9f8f6de9d5f7f3d52ac23e48ac51801f1dc950abe901da34d2a9e3baadb141a17c77ef3c565dd5ee5054b91cf63bb3c6ab83f72ab3aafe93d0fc3c2348b764fafb0b1c0733de51459aeab46580384bf9d74c4e28164b7cde247f891ba07891c9d872ad2bb")]
 
+[assembly: InternalsVisibleTo(@"Microsoft.TemplateEngine.Authoring.TemplateApiVerifier, PublicKey=0024000004800000940000000602000000240000525341310004000001000100f33a29044fa9d740c9b3213a93e57c84b472c84e0b8a0e1ae48e67a9f8f6de9d5f7f3d52ac23e48ac51801f1dc950abe901da34d2a9e3baadb141a17c77ef3c565dd5ee5054b91cf63bb3c6ab83f72ab3aafe93d0fc3c2348b764fafb0b1c0733de51459aeab46580384bf9d74c4e28164b7cde247f891ba07891c9d872ad2bb")]
```

Comment:
is it needed? don't see it being referenced

Reply:
`RunnableProjectConfig` is needed and it's internal.